src/System.Private.CoreLib/shared/System/Boolean.cs

```diff
@@ -15,6 +15,7 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.Versioning;
 
+#nullable enable
```

Comment:
Could you move this to before the `using` statements, to be consistent to what the tool now produces?

Reply:
ok